conan/cli/commands/audit.py

```diff
@@ -20,15 +20,22 @@ def _add_provider_arg(subparser):
     subparser.add_argument("-p", "--provider", help="Provider to use for scanning")
 
 
-def _parse_error_threshold(result, error_level):
-    """Mark the result as error if any of the vulnerabilities has a severity greater than or equal to"""
+def _parse_error_threshold(result: dict, error_level: float):
+    """Mark the result as error if any of the vulnerabilities has a severity greater than or equal to
+    """
     if error_level is not None:
-        for ref in result["data"]:
-            for edge in result["data"][ref]["vulnerabilities"]["edges"]:
-                preferredBaseScore = float(edge["node"]["cvss"].get("preferredBaseScore", 0.0))
-                if preferredBaseScore >= error_level:
-                    result["is_error_severity"] = True
-                    break
+        try:
+            error_level = float(error_level)
+            for ref in result["data"]:
+                for edge in result["data"][ref]["vulnerabilities"]["edges"]:
+                    preferred_base_score = float(edge["node"]["cvss"].get("preferredBaseScore", 0.0))
+                    if preferred_base_score >= error_level and "error" not in result:
+                        result.update({"error": {"details":
+                                "The conan audit command exceeded the threshold severity level."}})
+                        break
+        except ValueError:
```

Comment:
Not great for handling a ``False`` value, I think it can hide other problems.

Reply:
Reverted by the commit 46aa6c3